src/Symfony/Component/Form/Extension/Core/Type/TimeType.php

```diff
@@ -297,12 +298,12 @@ public function configureOptions(OptionsResolver $resolver)
             'choice_translation_domain' => false,
         ]);
 
-        $resolver->setDeprecated('model_timezone', function (Options $options, $modelTimezone): string {
+        $resolver->setNormalizer('model_timezone', function (Options $options, $modelTimezone): ?string {
             if (null !== $modelTimezone && $options['view_timezone'] !== $modelTimezone && null === $options['reference_date']) {
```

Comment:
Now that we throw an exception, could we relax this behavior by deriving the default value of `view_timezone` from `model_timezone` when `reference_date` is null and `model_timezone` is not? Similar to https://github.com/symfony/symfony/pull/32762 4.4 but here at 5.0 ?

Reply:
make sense to me, I'll rebase here once #32762 is merged

Reply:
done

Reply:
https://github.com/symfony/symfony/pull/32759#discussion_r307756094 ?

Reply:
Does anything not work as you would expect it to? Both your added test as well as my test pass. So I guess this works as expected, doesn't it?

Reply:
My suggestion is about do the same but for `view_timezone` option, e.i. by default it can be equal to `model_timezone` value if provided, or should we wait until 5.1 to add this feature?

Reply:
In other words: Now, if we provide the option `model_timezone` only, it will always throw an exception. That's what I'd like relax here by setting a lazy default value for `view_timezone`.

Reply:
Oh yes, that makes sense. For 4.4, this is not possible though as it would be a BC break. So the next release we can introduce the change will be 5.1 as a new feature.

Reply:
I opened #32805 so that we do not forget about it